master/reference/calicoctl/resources/networkset.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Network Set Resource (GlobalNetworkSet)
+---
+
+A network set resource (GlobalNetworkSet) represents an arbitrary set of IP subnetworks/CIDRs, allowing it
+to be matched by Calico policy.  Network sets are useful for applying policy to traffic
+coming from (or going to) external, non-Calico, networks.
+
+The metadata for each network set includes a set of labels.  When Calico is calculating the set of
+IPs that should match a source/destination selector within a
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) rule, it includes
```

Comment:
we probably should be explicit here and say "global network policy " and link to "global network policy", since network sets are global scoped resources.